### PR TITLE
add String() for FieldValue

### PR DESCRIPTION
**Pull request by lance6716**
close https://github.com/go-mysql-org/go-mysql/issues/821

### CODE REVIEW

File: client/client_test.go [hunk 1]
```diff
@@ -423,3 +423,71 @@ func (s *clientTestSuite) TestStmt_Trans() {
...
+		`1`, "'\x02'", `3`, `'4.50000'`, `6.7`,
+		`'2019-01-01'`, `'2019-01-01 01:01:01.123'`, `'2019-01-01 01:01:01.1234'`, `'01:01:01.12345'`, `2019`,
+		`'char'`, `'varchar'`, "'binary\x00\x00\x00\x00'", `'varbinary'`, `'blob'`, `'a'`, `'a,b'`, `'{"a": 1}'`,
+		`NULL`,
+	}
+	for i, v := range result.Values[0] {
+		require.Equal(s.T(), expected[i], v.String())
+	}
+
+	// test can directly use to build a SQL, through it's not safe in most cases
```
Comment (lance6716) [inline client/client_test.go:479]: @zhujintao do you think it's OK to use?

**Review by lance6716 — COMMENTED** [review]
@atercattus PTAL, not sure if you have spare time

File: client/client_test.go (hunk 1 above)
Comment (lance6716) [reply client/client_test.go:479]:
```suggestion
	// test can directly use to build a SQL, though it's not safe in most cases
```

File: client/client_test.go [hunk 2]
```diff
@@ -423,3 +423,71 @@ func (s *clientTestSuite) TestStmt_Trans() {
...
+		`'2019-01-01'`, `'2019-01-01 01:01:01.123'`, `'2019-01-01 01:01:01.1234'`, `'01:01:01.12345'`, `2019`,
+		`'cha\'r'`, `'varchar'`, "'binary\x00\x00\x00\x00'", `'varbinary'`, `'blob'`, `'a'`, `'a,b'`, `'{"a": 1}'`,
+		`NULL`,
+	}
+	for i, v := range result.Values[0] {
+		require.Equal(s.T(), expected[i], v.String())
+	}
+
+	// test can directly use to build a SQL, though it's not safe in most cases
+	sql := fmt.Sprintf("INSERT INTO field_value_test VALUES (%s)", strings.Join(expected, ","))
```
Comment (lance6716) [inline client/client_test.go:480]: @zhujintao do you think it's OK to use?

Comment (zhujintao) [reply client/client_test.go:480]: yes, it works very well

**Pull request merged**